pkgs/tools/security/clamav-unofficial-sigs.nix

```
@@ -0,0 +1,47 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper, clamav, rsync, coreutils, gnutar, curl, gzip, dnsutils, gnupg, glibc, gawk, diffutils, bind }:
+
+stdenv.mkDerivation rec {
+  pname = "clamav-unofficial-sigs";
+  version = "7.2";
+
+  src = fetchFromGitHub {
+    owner = "extremeshok";
+    repo = "clamav-unofficial-sigs";
+    rev = "${version}";
+    sha256 = "sha256-KZh8f4BB31b4ZBOrp4z/+TCqsppa2cyfteM3OUVv57A=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/etc/clamav-unofficial-sigs
+    mkdir -p $out/bin
+
+    cp clamav-unofficial-sigs.sh $out/bin
```

```suggestion
    cp clamav-unofficial-sigs $out/bin
```

Executables under /bin should not have extensions like .sh or .py.